platform/vs/sonic-version.dep

```diff
@@ -0,0 +1,2 @@
+#DPKG FRK
+$(SONIC_VERSION)_CACHE_MODE  := none
```

Comment:
There are a lots of .lock files generated under global cache folder without any .tgz files when building for VS. Is it relative to the dep file?

Feb 15 23:14:50 
Feb 15 23:14:50 [ DPKG ] Cache is not enabled for sonic_version.yml package

Build command:
make SONIC_CONFIG_BUILD_JOBS=1 INSTALL_DEBUG_TOOLS=y SONIC_DPKG_CACHE_METHOD=cache SONIC_DPKG_CACHE_SOURCE=/nfs/dpkg_cache target/sonic-vs.img.gz

Reply:
Yes, it is created once.